src/orders/README.md

```diff
@@ -206,6 +206,39 @@ This creates two secrets managed by kubernetes, named `atp-db-creds` and `atp-jk
 The kubernetes deployment manifest `atp-spring-jpa.yml` references these secrets and provides them to the runtime environment of the containers. The application now references the properties through the environment variables instead of reading them from a file. Alternatively the config files themselves could have been provided and mounted as volumes in to the container.
 
 
+```bash
+kubectl create secret generic monitoring-secret \
```

Comment:
WTH? Why makes mandatory include this secret? This basically breaks several use case usages of MuShop, breaking several labs, workshops that are already in use. This need to be something totally optional

Reply:
This is not a mandatory step.. purely optional. I will correct the doc.